docs/Configure.md

```diff
@@ -4,7 +4,7 @@
 
 * [Configuration file](#configuration-file)
   * [Changing the data directory](#changing-the-data-directory)
-  * [Splitting the configuration](#splitting-the-configuration)
+  * [Splitting the configuration](#splitting-the-configuration)\
```

Comment:
is that \ at the end a typo?

Reply:
fixed in https://github.com/ACINQ/eclair/pull/2613/commits/c00ae33f62d53d4d3710f7dc05a273993eb47657